parsl/tests/configs/globus_compute.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from parsl.config import Config
+from parsl.executors import GlobusComputeExecutor
+
+
+def fresh_config():
+
+    endpoint_id = os.environ.get("GLOBUS_COMPUTE_ENDPOINT",
+                                 "4b116d3c-1703-4f8f-9f6f-39921e5864df")
```

Comment:
While anyone could potentially run tests against our tutorial EP, I don't think we should suggest doing so.

Reply:
Good call. I've changed this to require GLOBUS_COMPUTE_ENDPOINT to be specified for tests. Failing with a KeyError should be clear enough that something is misconfigured.